native_client/javascript/Makefile

```diff
@@ -41,10 +41,6 @@ package.json: package.json.in
 		package.json.in > package.json && cat package.json
 
 npm-dev: package.json
-ifeq ($(findstring _NT,$(OS)),_NT)
-	# node-gyp@5.x behaves erratically with VS2015 and MSBuild.exe detection
-	$(NPM_TOOL) install node-gyp@4.x
-endif
```

Comment:
Nice.

Reply:
Should the node-gyp entry in devDependencies now be changed to just "5.x"?

Reply:
I think I did it, yes